modules/home/assets/js/components/home-screen.js

```diff
@@ -7,14 +7,17 @@ const HomeScreen = ( props ) => {
 		embedUrl = 'https://www.youtube.com/embed/icTcREd1tAg?si=40E8D1hdnu26-TXM';
 
 	return (
-		<Container sx={ { display: 'flex', flexDirection: 'column', gap: { xs: 1, md: 3 }, py: { xs: 2, md: 6 }, maxWidth: { md: '990px' } } }>
-			<TopSection
-				videoUrl={ videoUrl }
-				embedUrl={ embedUrl }
-				topData={ props.homeScreenData.data.top[ 0 ] }
-				createNewPageUrl={ props.homeScreenData.createNewPageUrl }
-			/>
-		</Container>
+		<Box maxWidth="xl" sx={ { pr: 1 } }>
+			{ /* Box wrapper around the Container is needed to neutralize wp-content area left-padding, that becomes too apparent on smaller screens, and ensure that Container is getting desired width*/ }
```

Comment:
Maybe?
It would like to keep it short and specific.

```suggestion
			{ /* The `<Box>` wrapper around the `<Container>` is needed as a correction for the `#wpcontent` `padding-left` styling */ }
```